tinyfatdb/__init__.py

```diff
@@ -5,3 +5,13 @@
 __author__ = """Carlos Velez"""
 __email__ = 'velez.carlos.a@gmail.com'
 __version__ = '0.1.0'
+
+from .databases import TinyFatDB
+from .tables import TinyFatTable
+from .models import TinyFatModel
+from .querysets import TinyFatQueryset
+
+import os
+MODELS_DIR = os.path.join(os.path.dirname(__file__), "dbs")
+if not os.path.exists(MODELS_DIR):
+    os.mkdir(MODELS_DIR)
```

Comment:
If this package is to be used through services such as pip, I believe that this should be changed. Hiding a databases models inside of our package does not seem to be the best solution going forward. I think it would be better suited if we saved the models in a place like `/var/` on Unix systems (I am not sure for Windows systems)